lib/dogma/rules.ex

```diff
@@ -28,6 +29,9 @@ defmodule Dogma.Rules do
     Application.get_env :dogma, :rule_set, @default_rule_set
   end
 
+  defp async_test_script(script, formatter, rule_set) do
```

Comment:
Doesn't actively do anything but putting it in the enum call looked like:
```elixir
Enum.map(&Task.async(fn -> test_script(&1, formatter, rule_set) end))
```
which just felt a bit heavy. But happy to change to this if you don't like async_test_script/3